pkgs/tools/security/sequoia/default.nix

```
@@ -54,27 +70,24 @@ rustPlatform.buildRustPackage rec {
     "build-release"
   ];
 
-  LIBCLANG_PATH = "${llvmPackages.libclang}/lib";
+  LIBCLANG_PATH = "${llvmPackages_10.libclang}/lib";
 
-  # Please check if this is still needed when updating.
-  # Exlude tests for sequoia-store, they often error with 'Too many open files' Hydra.
-  CARGO_TEST_ARGS = " --all --exclude sequoia-store";
+  # Sometimes, tests fail on CI (ofborg) & hydra without this
+  CARGO_TEST_ARGS = "--exclude sequoia-store";
```

```suggestion
  CARGO_TEST_ARGS = "--workspace --exclude sequoia-store";
```

According to [cargo docs](https://doc.rust-lang.org/cargo/commands/cargo-test.html), `--exclude` must be used with `--workspace` (even though it runs exactly the same tests for me regardless).

---

:+1:.